compiler/src/org.graalvm.compiler.nodes/src/org/graalvm/compiler/nodes/calc/ConditionalNode.java

```diff
@@ -76,6 +80,11 @@ public ConditionalNode(LogicNode condition, ValueNode trueValue, ValueNode false
         this.condition = condition;
         this.trueValue = trueValue;
         this.falseValue = falseValue;
+        // In order to be able to generate some platform-specific nodes,
+        // this Boolean variable records whether the platform is AArch64 or not.
+        if (Services.getSavedProperties().get("os.arch").equals("aarch64")) {
+            isAArch64 = true;
+        }
```

Comment:
Now AMD64 & LLVM also support the AbsNode, so this can be applied to all architectures

The general way though to add architecture-specific traits is through `LoweringProvider`